api/v1/Makefile.protoc

```diff
@@ -41,4 +42,7 @@ all:
 			--go-grpc_out=require_unimplemented_servers=false,paths=source_relative:. \
 			--go-json_out=orig_name=true,paths=source_relative:. \
 			$${proto}; \
+		$(PROTOC) $(HUBBLE_PROTOC_PLUGINS) -I $(HUBBLE_PROTO_PATH) \
```

Comment:
How does this work without passing in all the plugin parameters? Are protoc and the plugins ignoring any already present files?

Or asked differently - why do we need to pass in `$(HUBBLE_PROTOC_PLUGINS)` here? Wouldn't the following be sufficient?
```suggestion
		$(PROTOC) --plugin=$(GOPATH)/bin/protoc-gen-gotag -I $(HUBBLE_PROTO_PATH) \
```

On a side-note: I liked that the previous version had a commit why this is needed. I think it would be valuable to have a comment here that `gotag` needs the file to already exist

Reply:
You're right, only gotag plugin is needed in the second invocation. I changed it and also rebased the commit to include required image changes in this PR